app/src/main/java/dev/baseio/discordjetpackcompose/MainActivity.kt

```diff
@@ -29,9 +28,11 @@ class MainActivity : ComponentActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        installSplashScreen()
-        /**For Making the App FullScreen*/
+
+        setTheme(R.style.Theme_DiscordJetpackCompose)
```

Comment:
@aditya-bhawsar Can we move this above super.onCreate?

Reply:
done